### PR TITLE
Add integration test to MNB and MARE

### PR DESCRIPTION
**Pull request by KickItLikeShika**
Adding integration test to [MeanNormalizedBias](https://github.com/pytorch/ignite/blob/master/tests/ignite/contrib/metrics/regression/test_mean_normalized_bias.py) and [MeanAbsoluteRealtiveError](https://github.com/pytorch/ignite/blob/master/tests/ignite/contrib/metrics/regression/test_mean_absolute_relative_error.py)

Check list:

- [x] New tests are added (if a new feature is added)
- [ ] New doc strings: description and/or example code are in RST format
- [ ] Documentation is updated (if required)

### CODE REVIEW

**Comment by sdesrozis** [comment]
This metrics are not distributed. Why not do the distribution in this PR and add integration for distributed ?

@KickItLikeShika Waht do you think ?

**Comment by KickItLikeShika** [comment]
@sdesrozis most of the other metrics are not distributed as well, i think separating the PRs would be better. I can work on a separate PR for most of the metrics in `ignite.contrib.metrics.regression` and make them distributed, right?

File: tests/ignite/contrib/metrics/regression/test_mean_absolute_relative_error.py [hunk 1]
```diff
@@ -76,3 +78,46 @@ def test_zero_sample():
...
+        np_y_pred = y_pred.numpy()
+
+        data = list(range(y_pred.shape[0] // batch_size))
+        mare = engine.run(data, max_epochs=1).metrics["mare"]
+
+        abs_error = np.sum(abs(np_y - np_y_pred) / abs(np_y))
+        num_samples = len(y_pred)
+        sum_error = abs_error
+        sum_samples = num_samples
+        res = sum_error / sum_samples
```
Comment (sdesrozis) [inline tests/ignite/contrib/metrics/regression/test_mean_absolute_relative_error.py:106]:
```suggestion
        res = abs_error / num_samples
```
and remove `sum_samples` and `sum_error`

File: tests/ignite/contrib/metrics/regression/test_mean_normalized_bias.py [hunk 2]
```diff
@@ -72,3 +73,44 @@ def test_mean_error():
...
     assert m.compute() == pytest.approx(np_ans)
+
+
+def test_integration():
+    def _test(y_pred, y, batch_size):
+        def update_fn(engine, batch):
+            idx = (engine.state.iteration - 1) * batch_size
+            y_true_batch = np_y[idx : idx + batch_size]
+            y_pred_batch = np_y_pred[idx : idx + batch_size]
+            return idx, torch.from_numpy(y_pred_batch), torch.from_numpy(y_true_batch)
```
Comment (sdesrozis) [inline tests/ignite/contrib/metrics/regression/test_mean_normalized_bias.py:84]:
```suggestion
            return torch.from_numpy(y_pred_batch), torch.from_numpy(y_true_batch)
```

File: tests/ignite/contrib/metrics/regression/test_mean_normalized_bias.py [hunk 3]
```diff
@@ -72,3 +73,44 @@ def test_mean_error():
...
+
+        engine = Engine(update_fn)
+
+        m = MeanNormalizedBias(output_transform=lambda x: (x[1], x[2]))
```
Comment (sdesrozis) [inline tests/ignite/contrib/metrics/regression/test_mean_normalized_bias.py:88]:
```suggestion
        m = MeanNormalizedBias()
```

File: tests/ignite/contrib/metrics/regression/test_mean_absolute_relative_error.py [hunk 4]
```diff
@@ -76,3 +78,46 @@ def test_zero_sample():
...
         m.compute()
+
+
+def test_integration():
+    def _test(y_pred, y, batch_size):
+        def update_fn(engine, batch):
+            idx = (engine.state.iteration - 1) * batch_size
+            y_true_batch = np_y[idx : idx + batch_size]
+            y_pred_batch = np_y_pred[idx : idx + batch_size]
+            return idx, torch.from_numpy(y_pred_batch), torch.from_numpy(y_true_batch)
```
Comment (sdesrozis) [inline tests/ignite/contrib/metrics/regression/test_mean_absolute_relative_error.py:89]:
```suggestion
            return torch.from_numpy(y_pred_batch), torch.from_numpy(y_true_batch)
```

File: tests/ignite/contrib/metrics/regression/test_mean_absolute_relative_error.py [hunk 5]
```diff
@@ -76,3 +78,46 @@ def test_zero_sample():
...
+
+        engine = Engine(update_fn)
+
+        m = MeanAbsoluteRelativeError(output_transform=lambda x: (x[1], x[2]))
```
Comment (sdesrozis) [inline tests/ignite/contrib/metrics/regression/test_mean_absolute_relative_error.py:93]:
```suggestion
        m = MeanAbsoluteRelativeError()
```

**Comment by sdesrozis** [comment]
@KickItLikeShika Thank you ! I left comments

**Review by sdesrozis — APPROVED** [review]
LGTM !

**Pull request merged**